abl/util/lockfile.py

```diff
@@ -60,7 +60,8 @@ def __enter__(self):
             else:
                 fcntl.flock(self.file, lock_flags)
         except IOError, e:
-            if e[0] == errno.EAGAIN:
+            error_code = errno.EACCES if is_windows else errno.EAGAIN
```

Comment:
Found something interesting:

- on BSD fcntl.flock would be flock, which returns EWOULDBLOCK i.e. EAGAIN
cf https://www.freebsd.org/cgi/man.cgi?query=flock&sektion=2

- on POSIX, fcntl.flock would be implemented as fcntl, which can return EAGAIN or EACCESS
http://pubs.opengroup.org/onlinepubs/9699919799/functions/fcntl.html

- on Windows msvcrt.locking uses the _locking function, which returns EACCESS:
https://msdn.microsoft.com/fr-fr/library/8054ew2f(v=vs.100).aspx